### DISCUSSION TITLE
malware attack on GitHub

### DISCUSSION THREAD

**Discussion by shakfu** [community/community#28140] [Other Feature Feedback, Questions, & Ideas] [answered]
I just saw Stephen Lacy' Twitter post (<https://twitter.com/stephenlacy/status/1554697077430505473>) about a massive malware attack on GitHub using fake accounts which and ran a similar search "ovz1.j19544519" as his post and its seems these fake accounts are still active with the malware code.

Is there any response from Github about such a widespread attack? Where does one report such security threats typically?

**Comment by radar-blip** [comment]
Just leave this post up and maybe Github Staff will see it. I dont know where to report it.

**Comment by martinwoodward** [comment, marked as answer]
Hey folks 👋 - sorry for the concern. We've posted an initial response here:

https://twitter.com/githubsecurity/status/1554843443200806913

**Reply by radar-blip** [reply]
Thanks. Appreciate that!

**Reply by airtower-luna** [reply]
Thanks for the note! Might've been nice to quote/link the reporting tweet for credit (*not* mine, I want to emphasize). :slightly_smiling_face:

**Comment by shakfu** [comment]
@martinwoodward 

> * No repositories were compromised
> * Malicious code was posted to cloned repositories, not the repositories themselves
> * The clones were quarantined and there was no evident compromise of GitHub or maintainer accounts

Thanks for the response and clarification, especially the part about quarantining the clones with malicious code.
